Source/Core/Core/Hooks.cpp

```diff
@@ -0,0 +1,11 @@
+// Copyright 2016 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Core/Hooks.h"
+
+namespace Hook
+{
+HookableEvent<> ControllerHotplugEvent;
+HookableEvent<u32, u32, u32> XFBCopyEvent;
```

Comment:
What do you think about querying the hooks within a set, by name and argument types? So the ownership will be in Common, and everything already depends on Common.